app/mailers/hypertension_fast_track_pilot_mailer.rb

```diff
@@ -5,7 +5,7 @@ def build(submission)
     @submission = submission
 
     subject =
-      if submission.form_json.include? 'VAMC_Hypertension_Rapid_Decision_Evidence.pdf'
+      if submission.rrd_hypertension_claim_processed
```

Comment:
Add a `?` and make it more general:
```suggestion
      if submission.rrd_claim_processed?
```